src/Oro/ORM/Query/AST/Platform/Functions/Postgresql/Cast.php

```diff
@@ -27,6 +27,16 @@ public function getSql(SqlWalker $sqlWalker)
             return $timestampFunction->getSql($sqlWalker);
         }
 
+        /**
```

Comment:
Please, NOTE! This extensions are cross DB and them must not be strictly connected to some one. For example in future we may add support for Oracle, MS SQL or another DB